README.md

```diff
@@ -0,0 +1,46 @@
+# @sveltejs/branding
+
+Logos and Logotypes for Svelte in different formats.
+
+`svelte-logo.pdf` or `sapper-logo.pdf` are editable Adobe-Illustrator-CC-files which can be useful for occasions where none of the provided solutions fit.
+
+`colors.ase`  is the exported svelte color-palette [Adobe CC]
+
+**Don't** alter paths or proportions in any way.
+
+
+
+### Icons
+
+* Icons are available at sizes 16, 24, 32, 48, 64 or 128px
+* Either 1x or 4x resolution
+* .png or .svg
+
+<img style="width:32px; height:32px" src="https://github.com/sveltejs/branding/icons/svelte.svg" alt="svelte icon 32x32" />
```

Comment:
This won't render (see https://github.com/sveltejs/branding/tree/add-icons) maybe better to use the PNG.

Reply:
Ahh Ok. Didn't know that. Will change 'em.